lib/state_abbr.rb

```diff
@@ -11,6 +14,20 @@ def get_us_state_from_abbr(abbr, include_dc = false)
   return include_dc ? STATE_ABBR_WITH_DC_HASH[abbr] : STATE_ABBR_HASH[abbr]
 end
 
+# Returns the abbreviation for the supplied state name
+# @param [String|Symbol] name - full state name, e.g. "Washington"
+# @param [Boolean] include_dc - (default: false) Whether to include Washington DC as a state.
+# @returns [String] state abbreviation, or nil
+def get_us_state_abbr_from_name(name, include_dc = false)
```

Comment:
Would we ever want to exclude DC?

Reply:
Not that I know of, but the inverse method had that option so I added it for consistency